### DISCUSSION TITLE
Filter Users List

### DISCUSSION THREAD

**Discussion by mieliespoor** [grafana/grafana#40024] [General]
At the moment it is possible to search/filter the users list only by login, email or name. This make it impossible to get a list of users who are Admin or Editors or any other type of role.

A more advanced filter on this screen would make managing users a lot easier. For example, I would like to filter on the following fields/conditions:

- Role
- Last Seen
- Team
- Existing free text (login, email or name)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
